### ISSUE TITLE
Using translatable globals in notifications email address

### ISSUE THREAD

**Issue by wallacio** [verbb/formie#517]
I'd like to be able to create a site-specific value for admin notification email recipient, using a translatable global field in a mult-site installation.

As per #271 it's possible to specify a global field value as a recipient although this doesn't seem to respect a translatable field. The value substituted is that from the primary site. Submissions are being saved to the correct site, so Formie does seem to be site-aware.

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.9](https://github.com/verbb/formie/releases/tag/1.4.9)
